hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -144,6 +186,25 @@ public HoodieTestTable addCommit(String instantTime) throws Exception {
     return this;
   }
 
+  public HoodieCommitMetadata createCommitMetadata(WriteOperationType operationType, String commitTime, HoodieTestTableState testTableState) {
+    String actionType = MERGE_ON_READ.equals(metaClient.getTableType()) ? DELTA_COMMIT_ACTION : COMMIT_ACTION;
+    return createCommitMetadata(operationType, commitTime, Collections.emptyMap(), testTableState, false, actionType);
+  }
+
+  public HoodieCommitMetadata createCommitMetadata(WriteOperationType operationType, String commitTime,
+                                                   HoodieTestTableState testTableState, boolean bootstrap) {
+    String actionType = MERGE_ON_READ.equals(metaClient.getTableType()) ? DELTA_COMMIT_ACTION : COMMIT_ACTION;
```

Comment:
is nt there a direct method to get the commit action type?

Reply:
There is one actually: `CommitUtils#getCommitActionType`